reference_code/verifiers/circom/README.md

```diff
@@ -0,0 +1,3 @@
+## Circom Verifiers
```

Comment:
Note that I'm nesting two layers here: `verifiers/circom/solidity`.  I think the eventual structure we want to have is `verifiers/{framework}/{verifier_framework}`.